### PR TITLE
Adding host configurations for F# templates

### PR DESCRIPTION
**Pull request by jmarolf**
Adding host configurations so these templates can be picked up by the VS project UI

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "uiFilters": [ "oneaspnet" ],
+  "supportsDocker": true,
+  "ports": [
+    {
+      "name": "HttpPort",
+      "useHttps": false
+    },
+    {
+      "name": "HttpsPort",
+      "useHttps": false
```
Comment (phenning) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json]: should be true for HttpsPort

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  ],
+  "excludeLaunchSettings": false,
```
Comment (phenning) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json]: Technically this is not needed, VS will not recognize it and thus would never pass on this setting when creating the template. Additionally, VS always "requires" a launch settings file.

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-FSharp/.template.config/vs-2017.3.host.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    {
+      "name": "HttpPort",
+      "useHttps": false
+    },
+    {
+      "name": "HttpsPort",
+      "useHttps": true
+    }
+  ],
+  "excludeLaunchSettings": false,
```
Comment (phenning) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-FSharp/.template.config/vs-2017.3.host.json]: again, don't really need excludeLaunchSettings

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/.template.config/vs-2017.3.host.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "name": "HttpPort",
+      "useHttps": false
+    },
+    {
+      "name": "HttpsPort",
+      "useHttps": true
+    }
+  ],
+  "excludeLaunchSettings": false,
```
Comment (phenning) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/.template.config/vs-2017.3.host.json]: not needed

**Comment by jbagga** [comment]
@phenning Updated

**Comment by jbagga** [comment]
cc @Eilon Merging this in for release/2.1

**Pull request merged**